src/content/en/updates/2017/02/navigation-preload.md

```diff
@@ -0,0 +1,338 @@
+{# wf_featured_snippet: Navigation preload lets you overcome service worker startup time by making requests in parallel. #}
+
+# Speed Up Service Worker With Navigation Preloads {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+## TL;DR
+
+* In some situations, [service worker boot-up time can delay a network
+  response](#the-problem).
+* A new experimental feature, [navigation preload](#the-solution), fixes this by
+  allowing you to make the request in parallel with service worker boot-up.
+* You can distinguish preload requests from regular navigations using a header,
+  and [serve different content](#header).
+* Navigation preload is in Canary behind a flag, and the API may change in
+  response to developer feedback.
+* It'll remain behind a flag in Chrome stable 57 (likely to be released in
+  March), but you can [apply for an origin trial](#origin-trial) to test it with
+  real users.
+
+## The problem {: #the-problem }
+
+When you navigate to a site that uses a service worker to handle fetch events,
+the browser asks the service worker for a response. This involves booting up the
+service worker (if it isn't already running), and dispatching the fetch event.
+
+The bootup time depends on the device & conditions. It's usually around 50ms, on
+mobile it's more like 250ms, but in extreme cases (slow devices, CPU in
+distress) it can be over 500ms. However, since the service worker stays awake
+for a browser-determined time between events, you only get this delay
+occasionally, such as when the user navigates to your site from a fresh tab, or
+another site.
+
+The boot-up time isn't a problem if you're responding from the cache, as the
+benefit of skipping the network is greater than the boot-up delay. But if you're
+responding using the network…
+
+{% framebox height="38px" %}
+<style>
+  .group {
+    height: 38px;
+    display: flex;
+  }
+  .group > div {
+    display: flex;
+    align-items: center;
+    font: normal 1.2rem/1 sans-serif;
+    text-shadow: 0 1.6px 2.7px rgba(0,0,0,0.5);
+    color: #fff;
+    box-sizing: border-box;
+    padding: 0 8px;
+    white-space: pre;
+  }
+  .sw-startup {
+    margin-right: 1px;
+    width: 100px;
+    background: #f0c457;
+  }
+  .network-request {
+    flex: 1;
+    background: #6ea1e3;
+  }
+</style>
+<div class="group">
+  <div class="sw-startup">SW boot</div>
+  <div class="network-request">Network request</div>
+</div>
+{% endframebox %}
+
+…the network request is delayed by the service worker booting-up.
+
+We're continuing to reduce the boot-up time by [using code-caching in
+V8](http://v8project.blogspot.com/2015/07/code-caching.html), by [skipping
+service workers that don't have a fetch
+event](https://bugs.chromium.org/p/chromium/issues/detail?id=605844), by
+[launching serivce workers
+speculatively](https://codereview.chromium.org/2045153003), and other
+optimisations, but it'll always be greater than zero.
```

Comment:
Grammatically, the last thought doesn't need to be a sentence. I think it merits the emphasis that a separate sentence would give it. Consider changing.